src/settings/Api.ts

```diff
@@ -46,17 +46,8 @@ export class Api {
 
 	listGroups(): Thenable<Group[]> {
 		const version = parseInt(OC.config.version, 10);
-		if (version >= 14) {
-			return $.getJSON(OC.linkToOCS('cloud', 1) + 'groups/details')
-				.then((data: OCSResult<{ groups: Group[]; }>) => data.ocs.data.groups);
-		} else {
-			return $.getJSON(OC.linkToOCS('cloud', 1) + 'groups')
-				.then((data: OCSResult<{ groups: string[]; }>) => data.ocs.data.groups.map(group => {
-					return {
-						id: group,
-						displayname: group
-					};
-				}));
+		return $.getJSON(OC.linkToOCS('cloud', 1) + '/groups/details')
```

Comment:
I assume this failing due to the `@nextcloud/router` bump on server, right? So no backports needed.

Reply:
I think it is. At least I wasn't encountering this error on stable21, let me try to check stable22 just to be sure.

Reply:
Good that I checked it seems to fails on stable22 too, so will need to be backported to stable22. Do you maybe know why it now suddenly fails to build in the ci, locally it works?